Add README.md and improve onboarding instructions (#35)
quarkiverse/quarkiverse-wiki: README.md

> # Welcome to Quarkiverse!

Reviewer: shouldnt this just link to the wiki
and add this doc content tothe home.md page?

Author: Ah right I'll fix this ;-)

The Quarkiverse GitHub organization provides repository hosting (including build, CI and release publishing setup) for Quarkus extension projects contributed by the community.

You can browse the [Quarkiverse documentation](./docs/Home.md) to get more info.

## Here is the flow to get your extension onboarded:

> 1. The first step is to create an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) (make sure to check first if the extension you’re proposing is not created yet) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss if the proposal extension should belong to core or live in the Quarkiverse Hub.

Reviewer: ```suggestion
1. The first step is to create or pick an **Extension Proposal** ticket in the [Quarkus issue tracker](https://github.com/quarkusio/quarkus/issues) and let us know if you are interested in leading the development of this extension by commenting on the issue. The Quarkus team will review and discuss with you and community if the proposal extension should belong to core or live in the Quarkiverse Hub.
```

Reviewer: maybe ask to mention @quarkiverse-team ? (and create that team)

Author: @maxandersen we could automate that right? (pinging the right team)

Reviewer: Not sure what hint we would use to know when to add the cc  hence asking user to do it.

2. When decided, the Quarkiverse Team will initialize the repository in the Quarkiverse organization for you with skeleton code ready. Until then, if not already done, you can start implementing your extension from a skeleton with this command:
```shell
mvn io.quarkus:quarkus-maven-plugin:create-extension \
 -DgroupId=io.quarkiverse.[myext] -DextensionId=[my-ext] -U
```

> 3. Enable in the [Quarkus Ecosystem-CI](https://github.com/quarkusio/quarkus-ecosystem-ci)

Reviewer 2: What does that mean exactly?

Reviewer: ```suggestion
3. Follow instructions in [Quarkus Ecosystem-CI](https://github.com/quarkusio/quarkus-ecosystem-ci) to enable the extension to take part in overall testing.
```

Reviewer 3: I think it means the steps to have your extension to build against Quarkus main branch: https://github.com/quarkusio/quarkus-ecosystem-ci#what-do-i-need-to-do-as-a-quarkus-platform-participant
